### ISSUE TITLE
[cartocss]: marker z positioning

### ISSUE THREAD

**Issue by farfromrefug** [CartoDB/mobile-sdk#467]
<img width="602" alt="Screenshot 2021-10-18 at 12 56 51" src="https://user-images.githubusercontent.com/655344/137718176-def6c498-a428-4a38-b405-34269113f836.png">
As shown in the screenshot the z positioning of markers is wrong by default.
I know we have `placement-priority` but here what i want is the default placement we have with `Marker` where the z positioning is based on lat/long

The biggest issue with this is the wrong rendering while tilting.
<img width="315" alt="Screenshot 2021-10-18 at 12 59 09" src="https://user-images.githubusercontent.com/655344/137718446-a9a9a2f2-cfe0-4267-a4aa-506e95d6826c.png">

**Comment by mtehver** [comment]
Correct, this feature is missing when using CartoCSS markers. It requires per-frame sorting of markers which is currently not implemented in vector tile renderer. Currently the only workaround is to use `VectorLayer` with `Marker` instances.

I am looking into this, will likely add a new property that can be used to force sorting.
